docs/proposals/caching/README.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Caching member cluster resources for Karmada
+authors:
+reviewers:
+approvers:
+
+creation-date: 2022-02-14
+
+---
+
+# Caching member cluster resources for Karmada
+
+## Summary
+
+Provides a caching layer for Karmada to cache (member clusters) Kubernetes resources.
+
+## Motivation
+
+### Goals
+
+- Accelerate resource requests across regions
+- Provides a cross-cluster resource view
+- Compatible with (multi-cluster) multiple kubernetes resource versions
+- Uniform resource request entry
+- Reduce the pressure on the API server of the member cluster
+
+### Non-Goals
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1
+
+Imagine that we have a cross-regional application that needs to run in multiple regions (Beijing, Shanghai, Shenzhen) in order to provide service capabilities nearby. We achieve this by deploying a Kubernetes cluster in each region.
+
+Goals:
+
+- Get the distribution of this application in different clusters (resource aggregation)
+- (Quickly) get resource information of this application in different clusters (cross-regional resource access)
+
+#### Story 2
+
+Imagine we have a resource management platform, which consists of multiple microservices, each of which needs to frequently acquire, display, and operate resources from different clusters
+
+Goals:
+
+- Provide a (cross-cluster) operation entry for users (view resources, events, logs, and operate resources)
+- Reduce the call pressure on the member cluster API Server for high-frequency resource access
+- Provides a cloud-native cluster management capability to prevent each component from directly obtaining cluster credentials
+
+## Design Details
+
+### Cluster Management
+
+Determine the target cluster and the corresponding resource type of the cache through the member cluster
+
+Implementation: Watch CRD, respond to ADD (cache cluster)/UPDATE (update cached object)/DELETE (remove cached cluster and cache information) events
+
+### API
+
+Aggregation API:
+
+- Compatible with Kubernetes: Authentication, Authorization(RBAC), Access Control
```

Comment:
Do you mean the `search` API which is installed on `karmada-apiserver` and users use this API to get resources from cache?